qmail-pop3d.c

```diff
@@ -294,6 +302,7 @@ char **argv;
   sig_alarmcatch(die);
   sig_pipeignore();
  
+  if (!getuid()) die_root();
```

Comment:
getuid is defined in unistd.h, whouch should be included right below the sys/ headers.